docs/chaos-engineering/chaos-faults/aws/ec2-cpu-hog.md

```diff
@@ -0,0 +1,267 @@
+    <h2>Mandatory Fields</h2>
+    <table>
+        <tr>
+            <th> Variables </th>
+            <th> Description </th>
+            <th> Notes </th>
+        </tr>
+        <tr>
+            <td> EC2_INSTANCE_ID </td>
+            <td> ID of the target EC2 instance </td>
+            <td> For example: <code>i-044d3cb4b03b8af1f</code> </td>
+        </tr>
+        <tr>
+            <td> REGION </td>
+            <td> The AWS region ID where the EC2 instance has been created </td>
+            <td> For example: <code>us-east-1</code> </td>
+        </tr>
+    </table>
+    <h2>Optional Fields</h2>
+    <table>
+        <tr>
+            <th> Variables </th>
+            <th> Description </th>
+            <th> Notes </th>
+        </tr>
+        <tr>
+            <td> TOTAL_CHAOS_DURATION </td>
+            <td> The total time duration for chaos injection (sec) </td>
+            <td> Defaults to 30s </td>
+        </tr>
+        <tr>
+            <td> CHAOS_INTERVAL </td>
+            <td> The interval (in sec) between successive chaos injection</td>
+            <td> Defaults to 60s </td>
+        </tr>
+        <tr>
+            <td> AWS_SHARED_CREDENTIALS_FILE </td>
+            <td> Provide the path for aws secret credentials</td>
+            <td> Defaults to <code>/tmp/cloud_config.yml</code> </td>
+        </tr>
+        <tr>
+            <td> INSTALL_DEPENDENCIES </td>
+            <td> Select to install dependencies used to run the cpu chaos. It can be either True or False</td>
+            <td> Defaults to True </td>
+        </tr>
+        <tr>
+            <td> CPU_CORE </td>
+            <td> Provide the number of cpu cores to consume</td>
+            <td> Defaults to 0 </td>
+        </tr>
+        <tr>
+            <td> CPU_LOAD </td>
+            <td> Provide the percentage of a single cpu core to be consumed</td>
+            <td> Defaults to 100 </td>
+        </tr>
+        <tr>
+            <td> SEQUENCE </td>
+            <td> It defines sequence of chaos execution for multiple instance</td>
+            <td> Default value: parallel. Supported: serial, parallel </td>
+        </tr>
+        <tr>
+            <td> RAMP_TIME </td>
+            <td> Period to wait before and after injection of chaos in sec </td>
+            <td> Eg: 30 </td>
+        </tr>
+    </table>
+</details>
+
+## Experiment Examples
+
+### Common Experiment Tunables
+
+Refer the [common attributes](../common-tunables-for-all-experiments) to tune the common tunables for all the experiments.
+
+### CPU CORE
+
+It defines the cpu core value to be utilised on the EC2 instance. It can be tuned via `CPU_CORE` ENV.
+
+Use the following example to tune this:
+
+[embedmd]:# (./static/manifests/ec2-cpu-hog/cpu-core.yaml yaml)
+```yaml
+# cpu cores to utilize
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-nginx
+spec:
+  engineState: "active"
+  annotationCheck: "false"
+  chaosServiceAccount: ec2-cpu-hog-sa
```

Comment:
use the litmus-admin service account - we are not providing experiment specific rbac here